Berksfile

```diff
@@ -1,12 +1,12 @@
 source 'https://supermarket.chef.io'
 
 cookbook 'firewall', git: 'git@github.com:osuosl-cookbooks/firewall'
-cookbook 'nagios', git: 'git@github.com:osuosl-cookbooks/nagios'
+cookbook 'osl-nagios', git: 'git@github.com:osuosl-cookbooks/osl-nagios'
 # This is required because of the dependency in the nagios cookbook. This will
 # be removed when we switch to the newer upstream nrpe cookbook.
 cookbook 'osl-apache', git: 'git@github.com:osuosl-cookbooks/osl-apache'
 cookbook 'osl-munin', git: 'git@github.com:osuosl-cookbooks/osl-munin'
```

Comment:
Please remove ``osl-apache`` reference since we're removing ``osl-nagios``. Also remove the comment above (which you apparently didn't read :P )